_pages/attributes.md

```diff
@@ -177,6 +177,64 @@ Requires the `profile` or `profile:verified_at` scope.
 </td>
 <td markdown="1">
 `verified_at` (string, ISO8601 format)
+</td>
+    </tr>
+    -----------------------------
```

Comment:
Should we remove this line? I'm not seeing it rendered

Reply:
Yes, we can get rid of that-- I'd only put this in to keep my place and forgot to delete it afterwards.